src/test/resources/reverseTest/output/nt/00223/TT002234459.nt

```diff
@@ -1,94 +1,91 @@
-_:b0 <http://bibframe.org/vocab/agent> _:b1 .
-_:b0 <http://bibframe.org/vocab/relator> <http://id.loc.gov/vocabulary/relators/ctb> .
-_:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://bibframe.org/vocab/Contribution> .
-_:b1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://d-nb.info/standards/elementset/gnd#Person> .
-_:b1 <http://www.w3.org/2000/01/rdf-schema#label> "Kranz, Heinrich"^^<http://www.w3.org/2001/XMLSchema#string> .
+<http://lobid.org/resources/TT002234459#!> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/BibliographicResource> .
+<http://lobid.org/resources/TT002234459#!> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/lobid/lv#ArchivedWebPage> .
+<http://lobid.org/resources/TT002234459#!> <http://bibframe.org/vocab/contribution> _:b9 .
+_:b9 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b0 .
+_:b9 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b10 .
 _:b10 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b2 .
 _:b10 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b11 .
 _:b11 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b4 .
 _:b11 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b12 .
 _:b12 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b6 .
 _:b12 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/hasVersion> <http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=1638893&custom_att_2=simple_viewer> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/language> <http://id.loc.gov/vocabulary/iso639-2/deu> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/medium> <http://rdvocab.info/termList/RDACarrierType/1010> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/medium> <http://purl.org/ontology/bibo/Map> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/medium> <http://rdvocab.info/termList/RDACarrierType/1018> .
+<http://lobid.org/resources/TT002234459#!> <http://purl.org/dc/terms/subject> _:b13 .
 _:b13 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://d-nb.info/gnd/4049787-2> .
 _:b13 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b14 .
 _:b14 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://d-nb.info/gnd/4114067-9> .
 _:b14 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b15 .
 _:b15 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> _:b8 .
-_:b15 <http://www.w3.org/1999/02/22-rdf-syntax-ns#rest> _:b16 .
-_:b16 <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://dewey.info/class/914/> .
```

Comment:
You have to look up the original input files to understand what is going wrong [HT015082724](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/01508/HT015082724.nt), [HT013304490](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/01330/HT013304490.nt), [TT002234459](https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/input/nt/00223/TT002234459.nt)